src/parser/lib/EventRequestOptions.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Copyright 2025 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace LINE\Parser;
+
+/**
+ * Options for event request parsing.
+ */
+class EventRequestOptions
+{
+    /**
+     * @var callable|null Function that returns boolean to determine if signature validation should be skipped.
```

Comment:
It might be a good idea to explicitly specify that it returns a bool.
```suggestion
     * @var callable(): bool|null Function that returns boolean to determine if signature validation should be skipped.
```

Reply:
Thank you for the suggestion!
Yes, the intention is to allow dynamic control — the callable can return a boolean value to determine whether signature validation should be skipped.

However, we had to simplify the PHPDoc type to callable|null instead of callable(): bool|null, because phpDocumentor 3.x currently doesn’t support the newer callable syntax with return types.
To avoid breaking the documentation build, we decided to keep the simpler type declaration.

Reply:
<img width="1133" height="567" alt="image" src="https://github.com/user-attachments/assets/bec825a4-a861-49cf-b3b4-bf7c42d98b3b" />

https://docs.phpdoc.org/guide/references/phpdoc/types.html#:~:text=null%3B%0A%7D-,callable,-the%20element%20to

It says it's available from version 3.3.0, so is that not correct? We use 3.3.1 https://github.com/line/line-bot-sdk-php/blob/a227e57bbf4ccf1d7e7d8f76b43353cb0a32ebd1/.github/workflows/generate-code.yml#L56

Reply:
or ` null|callable(): bool`?